src/topology_utils.py

```diff
@@ -45,19 +49,33 @@ def get_auth_session(args):
     if args.key:
         key = args.key
 
-    session = requests.Session()
+    session = {}
+<<<<<<< HEAD
 
+=======
+    session['cert_reqs'] = 'CERT_REQUIRED'
+>>>>>>> master
```

Comment:
There's a couple of diff-looking lines that seemed to have snuck in here somehow. Can you clean them up?